pylasu/transformation/generic_nodes.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+
+from pylasu.model import Node
+
+
+@dataclass
+class GenericNode(Node):
```

Comment:
In an experiment I am working on (based on Kolasu), I add a sort of annotation on "normal" nodes to mark them as placeholder for something we do not know how to translate yet. In this way I can create a node of the type that fit the hierarchy. In Python this may be less of an issue, given it is dynamic.

Perhaps this generic node could contain a reference to the thing that we do not know how to translate. That could be useful for debugging

Reply:
Indeed the GenericNode has its origin set to the transformation source, this is ported exactly from Kolasu